.prettierrc.json

```
@@ -0,0 +1,7 @@
+{
+    "trailingComma": "es5",
+	"tabWidth": 4,
+	"useTabs": false,
+    "singleQuote": true,
+    "semi": false
```

This file contains a tabs and spaces. Easter egg? 🤣

```suggestion
    "trailingComma": "es5",
    "tabWidth": 4,
    "useTabs": false,
    "singleQuote": true,
    "semi": false
```

---

Sort alphabetically?

```suggestion
    "semi": false,
    "singleQuote": true,
    "tabWidth": 4,
    "trailingComma": "es5",
    "useTabs": false
```